hotness/consumers.py

```diff
@@ -283,18 +283,22 @@ def _handle_anitya_update(self, upstream, package, msg):
                 else:
                     note = 'Patching or scratch build for %s-%s failed.\n' % (package, version)
                     self.bugzilla.follow_up(note, bz)
-                    if 'logs' in rh_stuff['build_logs']:
-                        for log in rh_stuff['build_logs']['logs']:
-                            note = 'Build log %s.' % log
-                            self.bugzilla.attach_patch(log, note, bz)
+                    if 'build_logs' in rh_stuff and 'build_ref' in rh_stuff.get('build_logs'):
```

Comment:
Just a little quibble, but this could be simplified a bit, I think:
```
if 'build_ref' in rh_stuff.get('build_logs', {}):
```
This handles the possibility that 'build_logs' isn't a key in ``rh_stuff`` and provides a default iterable.